clients/src/main/java/org/apache/kafka/common/protocol/Errors.java

```diff
@@ -495,7 +496,14 @@ public ApiException build(String message) {
             public ApiException build(String message) {
                 return new OperationNotAttemptedException(message);
             }
-        });
+    }),
+    KAFKA_STORAGE_ERROR(56, "Disk error when trying to access log file on the disk.",
```

Comment:
Hmm, if we bump up the protocol, wouldn't we have the same problem on the old client using the old protocol? i.e, If the broker hits a KafkaStorageException serving a request from the old client, we have to convert KafkaStorageException to sth that the old client can recognize.